doxygen/dox/UsersGuide.dox

```diff
@@ -374,23 +374,23 @@ These documents provide additional information for the use and tuning of specifi
 </tr>
   <tr style="height: 49.00pt;">
   <td style="width: 234.000pt; border-bottom-style: solid; border-bottom-width: 1px; border-bottom-color: #228a22; vertical-align: top;padding-left: 6.00pt; padding-top: 3.00pt; padding-right: 6.00pt; padding-bottom: 3.00pt;">
-  <p style="font-style: italic; color: #0000ff;"><span><a href="http://www.hdfgroup.org/HDF5/doc/Advanced/DynamicallyLoadedFilters/HDF5DynamicallyLoadedFilters.pdf">HDF5 Dynamically Loaded Filters</a></span></p>
+  <p style="font-style: italic; color: #0000ff;"><span><a href="https://docs.hdfgroup.org/hdf5/rfc/HDF5DynamicallyLoadedFilters.pdf">HDF5 Dynamically Loaded Filters</a></span></p>
```

Comment:
This might be able to go to the Filters UG I just added last week in K5PLmodule.h
\page H5PL_UG The HDF5 Plugins